makefile

```diff
@@ -332,8 +332,8 @@ MFEM_DEFINES = MFEM_VERSION MFEM_VERSION_STRING MFEM_GIT_STRING MFEM_USE_MPI\
  MFEM_USE_SLEPC MFEM_USE_MPFR MFEM_USE_SIDRE MFEM_USE_CONDUIT MFEM_USE_PUMI\
  MFEM_USE_HIOP MFEM_USE_GSLIB MFEM_USE_CUDA MFEM_USE_HIP MFEM_USE_OCCA\
  MFEM_USE_CEED MFEM_USE_RAJA MFEM_USE_UMPIRE MFEM_USE_SIMD MFEM_USE_ADIOS2\
- MFEM_USE_MKL_CPARDISO MFEM_USE_AMGX MFEM_USE_MUMPS MFEM_SOURCE_DIR\
- MFEM_INSTALL_DIR
+ MFEM_USE_MKL_CPARDISO MFEM_USE_AMGX MFEM_USE_MUMPS MFEM_USE_CAMP \
+ MFEM_SOURCE_DIR MFEM_INSTALL_DIR
```

Comment:
Is there CAMP support in the CMake build system?

Reply:
No... I'll add that

Reply:
I did some testing and we don't actually need CAMP support in CMake if Umpire is built with its CAMP submodule.  This is also true with RAJA.

If RAJA or Umpire is built with an external CAMP the build fails unless `camp_DIR` is defined.